### ISSUE TITLE
Template: Contrast Issues - 1.4.3 Contrast (Minimum) 

### ISSUE THREAD

**Issue by syarwick** [stephenravenscraft/loyola-template-4.0#82]
The gray breadcrumbs text does not have a sufficient contrast ratio with the white background. #EEEEEE on #FFFFFF has a contrast ratio of 1.16:1.  

Additionally, the 'x' on the search modal is coming up as low contrast, when in fact it is on a dark background. This is technically fine but will be flagged on all automatic findings.
